WebContent/app/views/orders/list.jsp

```diff
@@ -21,13 +21,13 @@
   <nav>
     <ul class="nav-list">
       <li class="nav-item"><a href="pedido">Novo Pedido</a></li>
-      <li class="nav-item"><a href="#">Cliente</a></li>
-      <li class="nav-item"><a href="#">Vendedor</a></li>
+	  <li class="nav-item"><a href="clientes">Cliente</a></li>
```

Comment:
Prefira usar espaço em vez de tab para espaçamentos. Tab não tem tamanho fixo, e pode ficar desalinhado se for combinado com espaço.